### PR TITLE
Feat/sushi 1.0

### PR DESCRIPTION
**Pull request by noemide**
Restructured Repo to work with SUSHI 1.0 and later. See https://fshschool.org/docs/sushi/migration/

### CODE REVIEW

File: ig.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[IG]
+ig = fsh-generated/resources/ImplementationGuide-de.bbmri.fhir.json
+template = ./bbmri.deTemplate
```
Comment (alexanderkiel) [inline ig.ini]: newline :-)

File: input/pagecontent/5_support.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ While there are [multiple ways](https://www.hl7.org/fhir/validation.html) to val
 
 This IG and all dependencies are published as packages. This means you do not have to download anything manually (except the validator JAR), the vaidator will install the nescessary packages for you (usually in a *.fhir* folder in your home directory).
 
-**1. Download the latest JAR from [here](https://storage.googleapis.com/ig-build/org.hl7.fhir.validator.jar).** Using the latest version is important since bug-fixing updates happen regularly.
+**1. Download the latest JAR from [here](https://github.com/HL7/fhir-ig-publisher/releases).** Using the latest version is important since bug-fixing updates happen regularly.
```
Comment (alexanderkiel) [inline input/pagecontent/5_support.md]: Can't find a validator.jar there.

File: input/resources/parentCollection.StructureDefinition.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (alexanderkiel) [inline input/resources/parentCollection.StructureDefinition.json]: Aren't all the resources generated from Shorthand? If so, they should not be checked in.

File: fsh/config.yaml [hunk 4]
```diff
@@ -1,54 +0,0 @@
-# ╭─────────────────────────────ImplementationGuide-de.bbmri.fhir.json─────────────────────────────╮
```
Comment (alexanderkiel) [inline fsh/config.yaml:1]: Please move (rename) the config instead of deleting and recreating it.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: input/resources/parentCollection.StructureDefinition.json (hunk 3 above)
Comment (noemide) [reply input/resources/parentCollection.StructureDefinition.json]: Some (in fact, most) resources are from the pre-FSH era and therefore checked in as JSON. I might try to convert them to FSH using GoFSH, but figured that should be an extr PR.

File: fsh/config.yaml (hunk 4 above)
Comment (noemide) [reply fsh/config.yaml:1]: Done.

File: input/resources/parentCollection.StructureDefinition.json (hunk 3 above)
Comment (alexanderkiel) [reply input/resources/parentCollection.StructureDefinition.json]: Ok I see. But why are they all created newly? Didn't they exist before?

File: input/resources/Specimen.StructureDefinition.json [hunk 5]
```diff
@@ -54,28 +54,6 @@
         ],
         "mustSupport": true
       },
-      {
-        "id": "Specimen.extension:diagnosis.value[x].coding",
```
Comment (alexanderkiel) [inline input/resources/Specimen.StructureDefinition.json:58]: Why did you delete this?

Comment (noemide) [reply input/resources/Specimen.StructureDefinition.json:58]: Because this information is already stored in the extension, but for some reson this was the only extension where the contents were listed seperately, which lead to an error about conflicting paths (value[x].coding vs valueCoding) when combined with the new FSH definition of the extension.
